native/src/modules/feedback/Feedback.js

```diff
@@ -46,47 +48,87 @@ const Description = styled(ThemedText)`
   font-weight: bold;
 `
 
+const Heading = styled(ThemedText)`
+  font-size: 16px;
+  text-align: center;
+  padding: 10px 30px 30px;
+`
+
+const HappyIconContainer = styled.Image`
+  margin: 100px auto 10px;
+`
+
+const SadIconContainer = styled.Image`
+  margin: 0px auto 10px;
+`
+
 export type PropsType = {|
   comment: string,
   contactMail: string,
   sendingStatus: SendingStatusType,
   onCommentChanged: (comment: string) => void,
   onFeedbackContactMailChanged: (contactMail: string) => void,
-  isPositiveFeedback: boolean,
+  feedbackOrigin: FeedbackOriginType,
   onSubmit: () => Promise<void>,
   theme: ThemeType,
   t: TFunction
 |}
 
-const FeedbackModal = (props: PropsType) => {
+const Feedback = (props: PropsType) => {
   const renderBox = (): React.Node => {
-    const { theme, t, isPositiveFeedback, comment, contactMail, sendingStatus } = props
+    const { theme, t, feedbackOrigin, comment, contactMail, sendingStatus } = props
+    let title
+    let isSearchFeedback = false
+    switch (feedbackOrigin) {
```

Comment:
Maybe we could just let the parent component decide what the title is? I.e. we pass the (translated or untranslated) title key and whether it is positive or negative feedback to the component instead of doing the switch here? I mean feedback in the search because of missing content is also negative feedback, so maybe that would make sense? What do you think?

Reply:
I think in terms of dependency inversion principle it would really make sense to keep it like this. Otherwise the high-level components would depend on the low level components. Also in case of single-responsibility it would make sense to have all the code concerning displaying the feedback-stuff to be in Feedback.js.

Reply:
I don't really agree. Passing a title as a prop is just a customization to the `Feedback` component the way React works by passing props to child components imo, e.g. like setting custom styles, formats or texts. I don't really see a dependency of higher to lower level components here as well. Maybe the point is just that I don't like these switch statements if they are not completely necessary. I just think it is much more straight forward to pass the title directly instead of having to introduce a new feedback origin type and adjusting the switch in this component here when maybe introducing a new feedack type in the future (Open-Closed Principle).

But probably this is just a matter of preference. @Taggotty @maxammann what is your opinion on this?

Reply:
Probably the best way to conform to both Dependency Inversion and Open-Closed Principle would be to create a subclass for each FeedbackType, which is probably too complex in this context. 
Besides that I think it is cleaner to set the translations here, as they are in fact `feedback` translations, even though I also think that a switch case such as this is not the most flexible and inherently consistent approach (as the feedbackOrigin has to be conformant to isPositiveFeedback). Maybe instead of passing `feedbackOrigin` as a type you could infer the title by `isPositiveFeedback` and `routeType` instead. Maybe @maxammann also has an opinion on this 🤷🏼‍♀️

Reply:
so i will keep it, as it is.

Reply:
oke i kept the functionallity here, but removed switch case, instead using conditional rendering for title.